recipes/opencv/4.x/conanfile.py

```diff
@@ -21,6 +21,7 @@ class OpenCVConan(ConanFile):
         "contrib_freetype": [True, False],
         "contrib_sfm": [True, False],
         "parallel": [False, "tbb", "openmp"],
+        "with_ade": [True, False],
```

Comment:
We don't have a recipe for ade in conan-center-index (yet).
https://github.com/opencv/opencv/blob/68d15fc62edad980f1ffa15ee478438335f39cc3/modules/gapi/cmake/init.cmake#L13
We don't want cmake to download sources/binaries:
https://github.com/opencv/opencv/blob/68d15fc62edad980f1ffa15ee478438335f39cc3/modules/gapi/cmake/init.cmake#L19

Reply:
ade looks like a straightforward library to create a conan package for:
https://github.com/opencv/ade

Reply:
Thanks @madebr! will first create a new recipe for that then

Reply:
@madebr have added the new recipe in #6753 and have updated this recipe to use that

Reply:
no we have ade in cci